src/pages/discover.js

```diff
@@ -72,6 +106,40 @@ const DiscoverPage = ({ router }) => {
     });
   };
 
+  // const onLoadCollectives = (e, value) => {
+  // router.push({
+  // pathname: router.pathname,
+  // query: { ...router.query, offset: 0, show: value },
+  // });
+  // };
+  const getPledgedCards = gql`
+    query discover {
+      allCollectiveTags
+      allCollectives(isPledged: true, isActive: false) {
+        collectives {
+          id
+          description
+          slug
+          pledges: orders(status: PENDING) {
+            id
+            totalAmount
+            currency
+          }
+        }
+      }
+    }
+  `;
+
+  const pledgedCardData = graphql(getPledgedCards);
```

Comment:
This is not how you're supposed to use graphql, you're supposed to wrap the component. See the example I previously sent you: https://github.com/opencollective/opencollective-frontend/blob/96d0d92634c617a21b64495076cf42993d6641ab/src/apps/expenses/components/DownloadInvoicesPopOver.js#L205